src/java/org/apache/cassandra/index/sai/disk/v1/kdtree/BKDWriter.java

```diff
@@ -261,6 +266,7 @@ private long writeField1Dim(IndexOutput out, MutableOneDimPointValues reader,
     }
 
     // reused when writing leaf blocks
+    // these are only used for bytes, so endianness doesn't matter
```

Comment:
potential issue: what if they suddenly are not used for bytes only, and endianess starts to matter bacuase someone adds a call in another part of the codebase, without looking at this comment? How hard is it to substitute a correct data output with matching endianess here? Would it cause any perf degradation? I mean, if it doesn't take much time / doesn't increase complexity, I'd like to be on the safer side.